docs/.vuepress/theme/components/AposFeedback.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <!-- Based on https://github.com/ipfs/ipfs-docs-v2/blob/2a019d7c8abe99c21a3150752915ed278a4592ff/docs/.vuepress/theme/components/Feedback.vue -->
+  <div class="feedback">
+    <h3 class="feedback-heading" v-if="!voteSubmitted">
+      {{ titleTxt }}
+    </h3>
+    <div v-if="!voteSubmitted" class="feedback-actions">
+      <button class="feedback-button" :title="yesTxt"
+        v-on:click="sendFeedback(evtYes)"
+      >
+        <span>{{ yesTxt }}</span>
+      </button>
+      <button class="feedback-button ":title="noTxt"
+        v-on:click="sendFeedback(evtNo)"
+      >
+        <span>{{ noTxt }}</span>
+      </button>
+    </div>
+    <div v-if="thanksMessage" class="feedback-result">
+      <p>{{ thanksMessage }}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AposFeedback',
+  data: function() {
+    return {
+      voteSubmitted: false,
+      currentPath: this.$route.path,
+      thanksMessage: '',
+      thanksTxt: 'Thank you for your feedback.',
+      issueRequest: 'Please open a Github issue with the link below to let us know what should be improved.'
+    }
+  },
+  methods: {
+    sendFeedback: function(evtType) {
+      this.voteSubmitted = true;
+      this.showThanks(evtType);
+
+      if (!window.ga) {
+        return;
+      }
+
+      window.ga('send', 'event', {
+        eventCategory: evtType,
+        eventAction: 'click',
+        eventLabel: this.currentPath
+      });
+    },
+    showThanks: function (evtType) {
+      this.thanksMessage = evtType === 'unhelpful' ?
+        `${this.thanksTxt} ${this.issueRequest}` : this.thanksTxt;
+    }
+  },
+  watch: {
+    '$route.path': function(path) {
+      this.voteSubmitted = false;
+      this.thanksMessage = '';
+      this.currentPath = path;
+    }
+  },
+  props: {
+    titleTxt: {
+      type: String,
+      default: 'Was this information helpful?'
+    },
+    evtYes: {
+      type: String,
+      default: 'helpful'
+    },
+    evtNo: {
+      type: String,
+      default: 'unhelpful'
+    },
+    yesTxt: {
+      type: String,
+      default: 'Yes'
+    },
+    noTxt: {
+      type: String,
+      default: 'No'
+    }
+  }
+}
+</script>
+
+<style lang="stylus" scoped>
+@keyframes fadein {
```

Comment:
I don't feel like `scoped` makes a ton of sense here. That fadein is fairly utilitarion and this is actually on every page, right? Just riffing tho i don't actually care

Reply:
It's limited to this component, though.